content/docs/2.0/concepts/external-scalers.md

```diff
@@ -0,0 +1,582 @@
+    var payload = JsonConvert.DeserializeObject<USGSResponse>(await resp.Content.ReadAsStringAsync());
+
+    return new IsActiveResponse
+    {
+      // return true if there is more than 2 Earthquakes with mag > 1.0
+      Result = payload.features.Count(f => f.properties.mag > 1.0) > 2
+    };
+  }
+}
+```
+{{< /collapsible >}}
+
+{{< collapsible "Javascript" >}}
+`index.js`
+```js
+const grpc = require('grpc')
+const request = require('request')
+const externalScalerProto = grpc.load('externalscaler.proto')
+
+const server = new grpc.Server()
+server.addService(externalScalerProto.externalscaler.ExternalScaler.service, {
+  isActive: (call, callback) => {
+    const longitude = call.request.scalerMetadata.longitude
+    const latitude = call.request.scalerMetadata.latitude
+    if (!longitude || !latitude) {
+      callback({
+        code: grpc.status.INVALID_ARGUMENT,
+        details: 'longitude and latitude must be specified',
+      })
+    } else {
+      const now = new Date()
+      const yesterday = new Date(new Date().setDate(new Date().getDate()-1));
+
+      const startTime = `${yesterday.getUTCFullYear()}-${yesterday.getUTCMonth()}-${yesterday.getUTCDay()}`
+      const endTime = `${now.getUTCFullYear()}-${now.getUTCMonth()}-${now.getUTCDay()}`
+      const radiusKm = 500
+      const query = `format=geojson&starttime=${startTime}&endtime=${endTime}&longitude=${longitude}&latitude=${latitude}&maxradiuskm=${radiusKm}`
+
+      request.get({
+        url: `https://earthquake.usgs.gov/fdsnws/event/1/query?${query}`,
+        json: true,
+      }, (err, resp, data) => {
+        if (err) {
+          callback({
+            code: grpc.status.INTERNAL,
+            details: err,
+          })
+        } else if (resp.statusCode !== 200) {
+          callback({
+            code: grpc.status.INTERNAL,
+            details: `expected status 200, got ${resp.statusCode}`
+          })
+        } else {
+          // count how many earthquakes with mag > 1.0
+          let count = 0
+          data.features.forEach(i => {
+            if (i.properties.mag > 1.0) {
+              count++
+            }
+          })
+          callback(null, {
+            result: count > 2,
+          })
+        }
+      })
+    }
+  }
+})
+
+server.bind('127.0.0.1:9090', grpc.ServerCredentials.createInsecure())
+console.log('Server listening on 127.0.0.1:9090')
+
+server.start()
+```
+{{< /collapsible >}}
+
+<br />
+
+#### 4. Implementing `StreamingIsActive`
+
+Unlike `IsActive`, `StreamingIsActive` is called once when KEDA reconciles the `ScaledObject`, and expects the external scaler to push `IsActiveResponse` whenever the scaler needs KEDA to activate the deployment.
+
+This implementation creates a timer and queries USGS APIs on that timer, effictivly ignoring `pollingInterval` set in the scaledObject. Alternativly any other asynchronus event can be used instead of a timer, like an HTTP request, or a network connection.
```

Comment:
```suggestion
This implementation creates a timer and queries USGS APIs on that timer, effectively ignoring `pollingInterval` set in the scaledObject. Alternatively any other asynchronous event can be used instead of a timer, like an HTTP request, or a network connection.
```